packages/react/src/primitives/shared/utils.ts

```diff
@@ -27,18 +22,14 @@ export const getConsecutiveIntArray = (start: number, end: number) => {
 
 /**
  * Create a uuid to use with amplify fields unless
- * an id is already defined
- * @param id user specified id or undefined
+ * an id is provided
+ * @param id user specified id
  * @returns string
  */
-export const useAmplifyFieldID = (id?: string) => {
-  return React.useMemo(() => {
-    if (id) {
-      return id;
-    }
-    return `amplify-field-${nanoid()}`;
-  }, [id]);
-};
+export const useAmplifyID = (id?: string) => useId(id);
+
+// Prefixed autogenerated id created by useAmplifyID
+export const AUTO_GENERATED_ID_PREFIX = 'radix-id-';
```

Comment:
If I understand correctly, this variable is used to find elements IDs that matches the prefix generated by `useId`, right? 

https://github.com/radix-ui/primitives/blob/d3f1d5839888110939e61a0ccea6c6944205bb90/packages/react/id/src/id.tsx#L43

Reply:
Yes, that's correct. I colocated it with `useAmplifyID` so that it would be clearer. Should I add an additional comment?